src/solver/conv_asm_implicit_gemm_gtc_bwd.cpp

```diff
@@ -272,10 +266,10 @@ bool ConvAsmImplicitGemmGTCDynamicBwdXdlops::IsApplicable(const ConvolutionConte
 
     if(ctx.group_counts != 1)
         return false;
-    std::string kernel_name;
-    int block_size;
-    int grid_size;
-    return FindImplicitGemmGtcDynamicBwdKernel(ctx, kernel_name, block_size, grid_size);
+    bool isValid;
+    std::tie(isValid, std::ignore, std::ignore, std::ignore) =
+        FindImplicitGemmGtcDynamicBwdKernel(ctx);
+    return isValid;
```

Comment:
[Informative] This kind of design confuses me even more. The previous version of the code was clearer and, at least, did not return unused values in case of an error. However, I don't mind leaving code like that if you like it. I'm already used to the ugly code.